components/director/internal/domain/healthcheck/repository.go

```diff
@@ -0,0 +1,13 @@
+package healthcheck
+
+import (
+	"github.com/kyma-incubator/compass/components/director/internal/model"
+)
+
+type Repository struct {
```

Comment:
do we have to export this type? Maybe we should call it `inMemoryRepository` because when we introduce real repository, we may want to keep both repos (in memory and sql) at the same time for some time.